build/get_env_shell.py

```diff
@@ -81,16 +81,12 @@ def main():
     cmd += ["--user", f"{os.getuid()}:{os.getgid()}"]
 
     target_platform = utils.get_target_platform()
-    # if the environment variable is not set, don't pass `--platform` argument,
+    # If the environment variable is not set, don't pass `--platform` argument,
     # as it's not supported on some docker build environment.
-    if os.getenv("TARGET_PLATFORM") is not None and os.getenv(
-            "TARGET_PLATFORM") != "":
-        cmd += ["--platform", f"linux/{os.getenv('TARGET_PLATFORM')}"]
+    if target_platform.from_env:
+        cmd += ["--platform", f"linux/{target_platform.platform}"]
     else:
-        cmd += ["--env", f"TARGET_PLATFORM={target_platform}"]
-
-    if target_platform == "arm64":
-        cmd += ["--env", "ETCD_UNSUPPORTED_ARCH=arm64"]
```

Comment:
@STRRL I found a strange env here, and I removed it because we have any strong associations with etcd.

Reply:
I forgot we must set this env for testing on the arm64 machine. Re-added it in https://github.com/chaos-mesh/chaos-mesh/pull/4172/commits/d3ce18d2764c1f4855e91c1f9f3c55f6f1e304c1.